doc/guide/packages.xml

```diff
@@ -252,9 +252,33 @@ $ cockpit-bridge --packages
 }
 </programlisting>
 
-    <para>A file called <filename>override.json</filename> may be placed next to the manifest.
-      containing overrides to the information in the manifest. These override files are in the
-      simple <ulink url="https://tools.ietf.org/html/rfc7386">JSON Merge Patch</ulink> format.</para>
+  </section>
+
+  <section id="package-manifest-override">
+    <title>Manifest overrides</title>
+
+    <para>A file called <filename>override.json</filename> may be placed next to the manifest, or a file
+      <filename>&lt;name&gt;.override.json</filename> may be placed into <filename>/etc/cockpit/</filename>
```

Comment:
One thing that could maybe use some clarification (since we complicate this): is this according to the package name (as listed in the manifest.json) or per the name of the containing directory?

Reply:
I clarified the documentation for now to point out that it is the directory name, so that the PR is at least self-consistent. Overall this seems a little less confusing to me, but I'm open to the approaches above.

Reply:
I think the directory name is the correct approach.  I just think that it should be documented.  Saying "name" is a bit confusing, particularly because the property in the manifest is also called "name".

Incidentally: override files could actually rename packages, which is weird... but hey... you get to keep both pieces. :)

Reply:
From IRC discussion: yes, we want package directory here. The go-to use case for `name:` is "packagekit" and "ostree" both providing "name: updates", and the latter has a higher priority. We need to be clear which packae directory gets overridden.